tsconfig.json

```diff
@@ -2,12 +2,13 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "target": "ES2021",
-    "rootDir": "src",
```

Comment:
This has to be changed in a new PR as we are changing the build directory, it's breaking the deployment flow as it creates `server.js` in different directory than we expect/used to be which is inside `build` directory, so we can run `node build/server.js`

Reply:
We've to change command in `package.json` as well [here](https://github.com/shardeum/archive-server/blob/23e6cbb6a32d32eebca5ee1a27f5b91b34160d98/package.json#L18) `npm run start` command expects `server.js` to be inside `build` directory